src/deserialization.hpp

```diff
@@ -105,7 +106,7 @@ template <class TensorProtoDeserializator>
 Status deserializePredictRequest(
     const tensorflow::serving::PredictRequest& request,
     const tensor_map_t& inputMap,
-    InferenceEngine::InferRequest& inferRequest) {
+    InferenceEngine::InferRequest* inferRequest) {
```

Comment:
Why pointer not reference? Reference defends you in most cases against nullptr. With such interface in theory you should check for nullptr beforehand.